api/src/opentrons/protocol_api/core/engine/labware.py

```diff
@@ -91,10 +100,65 @@ def get_parameters(self) -> LabwareParametersDict:
     def get_quirks(self) -> List[str]:
         return self._definition.parameters.quirks or []
 
+    def _get_offset_location(self) -> LabwareOffsetLocation:
+        slot = self.get_deck_slot()
+        if None is slot:
```

Comment:
nip but can we do this instead?
```suggestion
        if slot is None:
```